docs/core/ibc.md

```diff
@@ -0,0 +1,347 @@
+<!--
+order: 9
+-->
+
+# IBC
+
+This document serves as a guide for developers who want to write their own IBC applications. Due to the modular design of the IBC protocol, IBC application developers do not need to concern themselves with the low-level details of clients, connections, and proof verification. Nevertheless a brief explanation of the lower levels of the stack is given so that application developers may have a high-level understanding of the IBC protocol. Then the document goes into detail on the abstraction layer most relevant for application developers (channels and ports), and describes how to define your own custom packets, and `IBCModule` callbacks.
```

Comment:
can you break up these lines in the whole doc? Opening this in vim makes it harder to read as everything is on one very long line. Also github suggestions don't work because there is too much for github to recognize the individual changes